packages/paste-website/src/components/component-status/PackageStatus.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+
+interface PackageStatusProps {
+  status: string;
+}
+
+const PackageStatus: React.FC<PackageStatusProps> = ({status}) => {
+  if (status === 'yes') {
+    return (
+      <Box display="flex" alignItems="center" justifyContent="center">
+        <SuccessIcon
+          display="inline-block"
+          decorative={false}
+          title="Design assets done"
```

Comment:
This should say something about the packages status, no?

Reply:
Yep. Fixed.